internal/workflow/activities/storage_test.go

```diff
@@ -0,0 +1,159 @@
+package activities_test
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/google/uuid"
+	"go.artefactual.dev/tools/mockutil"
+	"go.artefactual.dev/tools/ref"
+	"go.artefactual.dev/tools/temporal"
+	temporalsdk_activity "go.temporal.io/sdk/activity"
+	temporalsdk_testsuite "go.temporal.io/sdk/testsuite"
+	"go.uber.org/mock/gomock"
+	"gotest.tools/v3/assert"
+
+	goastorage "github.com/artefactual-sdps/enduro/internal/api/gen/storage"
+	storage_fake "github.com/artefactual-sdps/enduro/internal/storage/fake"
+	"github.com/artefactual-sdps/enduro/internal/workflow/activities"
+)
+
+func TestCreatePackageActivity(t *testing.T) {
+	t.Parallel()
+
+	type test struct {
+		name      string
+		params    *activities.CreateStoragePackageActivityParams
+		mockCalls func(m *storage_fake.MockClientMockRecorder)
+		want      *activities.CreateStoragePackageActivityResult
+		wantErr   string
+	}
+	for _, tt := range []test{
+		{
+			name: "Creates a new package",
+			params: &activities.CreateStoragePackageActivityParams{
+				Name:       "Package 1",
+				AIPID:      "88e5a5fc-5c74-40e8-aa50-d6542b45f251",
+				ObjectKey:  "3776a8f2-5ad5-4b4e-80c7-a888d9229cc1",
+				Status:     "stored",
+				LocationID: "92c3934d-8911-4ce4-8a44-ffa72a0b5720",
+			},
+			mockCalls: func(m *storage_fake.MockClientMockRecorder) {
+				m.Create(
+					mockutil.Context(),
+					&goastorage.CreatePayload{
+						Name:       "Package 1",
+						AipID:      "88e5a5fc-5c74-40e8-aa50-d6542b45f251",
+						ObjectKey:  "3776a8f2-5ad5-4b4e-80c7-a888d9229cc1",
+						Status:     "stored",
+						LocationID: ref.New(uuid.MustParse("92c3934d-8911-4ce4-8a44-ffa72a0b5720")),
```

Comment:
Same. I doubt it's noticeable, but it could be at some point with so many tests parsing UUIDs all the time.